### PR TITLE
Fix Espresso tests to fit new login view

### PR DESCRIPTION
**Pull request by jesmrec**
Changes included:

- Login button is now hidden till the URL is accepted. So, changed `isEnabled()` to `isDisplayed()`
- Add `context` as parameter of `FileDataStorageManager`
- return empty object for `OCCapability` till it is mocked using new networking stuff

### CODE REVIEW

File: androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java [hunk 1]
```diff
@@ -271,7 +271,7 @@ public void test3_check_login()
         SystemClock.sleep(WAIT_INITIAL_MS);
 
         // Check that login button is disabled
```
Comment (davigonz) [inline androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java]: You can update this comment to `Check that login button is hidden`

File: androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java [hunk 2]
```diff
@@ -344,7 +344,7 @@ public void test5_check_login_special_characters()
         Log_OC.i(LOG_TAG, "Test Check Login Special Characters Start");
 
         // Check that login button is disabled
```
Comment (davigonz) [inline androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java]: Same here and with other comments below

File: androidTest/java/com/owncloud/android/authentication/SAMLAuthenticatorActivityTest.java [hunk 3]
```diff
@@ -186,7 +187,7 @@ public void test1_check_login_saml()
         SystemClock.sleep(WAIT_INITIAL_MS);
 
         // Check that login button is disabled
```
Comment (davigonz) [inline androidTest/java/com/owncloud/android/authentication/SAMLAuthenticatorActivityTest.java]: Same here

**Comment by davigonz** [comment]
@jesmrec I suggested a couple of changes

File: androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java (hunk 1 above)
Comment (jesmrec) [reply androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java]: Done!

File: androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java (hunk 2 above)
Comment (jesmrec) [reply androidTest/java/com/owncloud/android/authentication/AuthenticatorActivityTest.java]: Done!

File: androidTest/java/com/owncloud/android/authentication/SAMLAuthenticatorActivityTest.java (hunk 3 above)
Comment (jesmrec) [reply androidTest/java/com/owncloud/android/authentication/SAMLAuthenticatorActivityTest.java]: Done!

**Comment by jesmrec** [comment]
Changes done!

**Comment by davigonz** [comment]
Great, code approved

**Pull request merged**